src/azure-cli/azure/cli/command_modules/feedback/custom.py

```diff
@@ -720,13 +720,6 @@ def _prompt_issue(recent_command_list):
 
     print(prefix)
 
-    # open issues page in browser and copy issue body to clipboard
-    # import pyperclip
-    # try:  # todo: if no longer using clipboard, remove dependency
-    #     pyperclip.copy(original_issue)
-    # except pyperclip.PyperclipException as ex:
-    #     logger.debug(ex)
```

Comment:
@adewaleo, could you weigh in on this change?

Reply:
it was the only reference to pyperclip in the whole codebase, so I just removed it, if it's still valuable, let me know

Reply:
I think this was the right thing to do, @jonringer. Just wanted to run this by @adewaleo to see if he had any latent plans to keep this. In general, I'm all about trying to reduce the number of dependencies that we have. Especially so if we have extraneous ones like this.

Reply:
I'm going to go ahead and call this good. @adewaleo and I can sync-up offline if there's any trouble.